src/components/Layout/Header/NavBar/MobileNavLink.tsx

```diff
@@ -1,10 +1,19 @@
+import type { ButtonProps } from '@chakra-ui/react'
 import { Button, Flex } from '@chakra-ui/react'
 import { memo, useCallback, useMemo } from 'react'
 import { useTranslate } from 'react-polyglot'
 import { Link as ReactRouterLink, matchPath, useLocation } from 'react-router-dom'
-import type { Route } from 'Routes/helpers'
 
-export const MobileNavLink = memo(({ label, shortLabel, path, icon, priority }: Route) => {
+type MobileNavLinkProps = {
+  label: string
+  shortLabel?: string
+  path: string
+  icon?: JSX.Element
+  order?: number
```

Comment:
Removed priority (which was a pass-through) added `order` which matches the `order` prop below and is *not* exactly matching the `priority` - it shouldn't - see consumption